test/utils/objects.go

```diff
@@ -85,6 +88,36 @@ func PrepareTestObjects(ctx context.Context, c client.Client, namespacePrefix st
 	return o
 }
 
+// CleanupTestObjects take care to remove the finalizers of the secret and configmaps
+func CleanupTestObjects(ctx context.Context, o *TestObjects) {
```

Comment:
Can you make this a receiver func for `TestObjects`?
```suggestion
func (o *TestObjects) Cleanup(ctx context.Context) {
```